consensus/cryptarchia-engine/src/lib.rs

```diff
@@ -167,7 +167,7 @@ where
     }
 
     pub fn fork_choice(&self) -> Branch<Id> {
-        let k = self.config.security_param as u64;
+        let k = self.config.security_param.get() as u64;
```

Comment:
Not a problem here, but this is what my more pedantic Clippy tells me about this:

> casts from `u32` to `u64` can be expressed infallibly using `From`
an `as` cast can become silently lossy if the types change in the future`

Reply:
Nice catch by pedantic Clippy. Fixed.